README.md

```diff
@@ -286,6 +287,16 @@ Detects changed files that have not yet been committed. This does **not** includ
 
 Detects files that are not yet tracked by git (see `git status` manual). This does **not** include _uncommitted_ files. A new file is not _untracked_ anymore after it is added to the index.
 
+### gib.excludePathRegex
+
+Can be used to exclude certain changed files from being detected as changed, reducing the number of modules to build.
+
+The regular expression does _not_ need to describe the entire (absolute) path, but only the relevant part _inside_ the git repository context. Example:
+```
+/tmp/repo/blacklisted/some-file.txt
+```
+will be excluded when using `-Dgib.excludePathRegex=blacklisted` or `-Dgib.excludePathRegex=some-file\..*` etc., but is _not_ excluded when adding to the regular expression anything _outside_ of the git repository context like `/tmp/repo` or `repo`.
```

Comment:
May be good to add example with multi regexes via pipe switching character pipe (|). https://www.regular-expressions.info/alternation.html

Reply:
one of the examples may be very common matching of some binary files like images eg ".*\.jpg"

Reply:
Good points, going to add this later.

Reply:
> one of the examples may be very common matching of some binary files like images eg ".*.jpg"

Come to think of it, binaries **do** impact the build outcome. Imagine a developers updates a jpg that normally ends up in the application deployment file. With GIB and such an exclusion, GIB would omit the respective module (in case there no other non-excluded changes in that module) and the application deployment file would still contain the old jpg.

Reply:
They do impact build, but usually not the tests. So if you just need to run the tests then you don't need to look at jpegs.